obspy/io/zmap/core.py

```diff
@@ -137,6 +137,8 @@ def _serialize(self, catalog):
             origin = ev.preferred_origin()
             if origin:
                 dec_year = self._decimal_year(origin.time)
+                dec_second = origin.time.second + \
+                    origin.time.microsecond / float(1e6)
```

Comment:
super minor nitpick: no need to wrap this in `float()` as the exponential notation will always produce floating point numbers.